recipes/extra-cmake-modules/all/conandata.yml

```diff
@@ -11,3 +11,9 @@ sources:
   "5.93.0":
     url: "https://download.kde.org/stable/frameworks/5.93/extra-cmake-modules-5.93.0.tar.xz"
     sha256: "093dea7b11647bc5f74e6971d47ef15b5c410cba2b4620acae00f008d5480b21"
+  "5.106.0":
```

Comment:
Do you need the version 5.106.0? Otherwise, I would ask to keep only 5.108.0 to build extra not used packages. In case someone need that specific version, we add later.

Reply:
removed